eng/pipelines/corefx-base.yml

```diff
@@ -0,0 +1,134 @@
+parameters:
+  # Required: Name to display for this job in the UI
+  displayName: ''
+  
+  # Required: Name for this specific job -- should be unique per each job.
+  name: ''
+
+  # Required: Pool to run the job on.
+  pool: {}
+
+  # Required: TargetOS value to know what script commands to use
+  # Accepted values:
+  # - Windows_NT
+  # - Linux
+  # - OSX
+  # - FreeBSD
+  targetOS: ''
+
+  # Required: Strategy matrix (the matrix that we're going to build for this job)
+  # The following variables should be defined within the matrix:
+  #   matrix:
+  #     matrix_display_name:
+  #      _configuration: Debug | Release
+  #      _architecture: x64 | x86 | arm | arm64
+  #      framework: (netcoreapp, netfx, uap, etc).
```

Comment:
why not `_framework`?

Reply:
I will update to be consistent and have all with a common prefix: https://github.com/dotnet/corefx/pull/33821#discussion_r238891696 -- thanks!